tests/helper.py

```diff
@@ -129,3 +138,32 @@ def operator_unitary_test(
         if the adjointness property does not hold
     """
     torch.testing.assert_close(u, operator.adjoint(operator(u)[0])[0], rtol=relative_tolerance, atol=absolute_tolerance)
+
+
+def autodiff_test(
+    operator: Operator[*tuple[torch.Tensor, ...], tuple[torch.Tensor, ...]],
+    *u: Any,
```

Comment:
why is this any?

Reply:
shoulnt this match *tuple[torch.Tensor, ...] from operator?

Reply:
i will try if that works

Reply:
also, this is not py310 compatible, we would have to use Unpack instead of splitting with *